File: exo2/main.java
class Main {
    public static void main(String[] args) {
    newFurniture1 = new Furniture("Knoll", "Marcel Breuer", 1925)
    System.out.print(newFurniture1.editor, newFurniture1.designer, newFurniture1.conceptionYear)
    }
}

class Furniture {
    String editor;
    String designer;
    int conceptionYear;

    Furniture(String editor, String designer, int conceptionYear){
        this.editor = editor;
        this.designer = designer;
        this.conceptionYear = conceptionYear;
    }
}

File: exo2/main.ts

class Furniture {
    editor: string;
    designer: string;
    conception_year: number;

    constructor(editor: string, designer: string, conception_year: number) {
        this.editor = editor,
            this.designer = designer,
            this.conception_year = conception_year;
    }
}

let wassily_chair = new Furniture("Knoll", "Marcel Breuer", 1925)

console.log(wassily_chair);
